src/andromede/model/model.py

```diff
@@ -170,13 +186,89 @@ def get_all_constraints(self) -> Iterable[Constraint]:
         )
 
 
+class ModelSelectionStrategy(ABC):
+    """
+    Abstract class to specify the strategy of the created problem.
+    Its derived classes select variables and constraints for the optimization problems:
+        - InvestmentProblemStrategy: Keep investment and coupling variables and constraints only for a BendersDecomposed master
+        - OperationalProblemStrategy: Keep operational and coupling variables and constraints only for a BendersDecomposed sub-problems
+        - MergedProblemStrategy: Keep all variables and constraints
+    """
+
+    @classmethod
```

Comment:
It would have been more "classic" from object oriented design point of view, maybe simpler, to use instance methods and pass instances to methods, not the class themselves.

Also I would rather move this to "simulation" package because it's part of the computation logic, not part of the definition of models.

Reply:
Done